addons/l10n_sg_ubl_pint/__manifest__.py

```diff
@@ -0,0 +1,17 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+{
+    'name': 'Singapore - UBL PINT',
+    'countries': ['sg'],
+    'version': '1.0',
+    'category': 'Accounting/Localizations/EDI',
+    'description': """
+    The UBL PINT e-invoicing format for singapore is based on the Peppol International (PINT) model for Billing.
+    """,
+    'depends': ['account_edi_ubl_cii'],
+    'data': [
+        'views/account_tax_views.xml',
+        'data/sg_ubl_templates.xml',
+    ],
+    'installable': True,
```

Comment:
Just to be sure no auto install needed ?
EDIT: "This is not meant to be used directly, but rather to be extended by country-specific modules." i suppose it's normal then

Reply:
Hm I didn't set the other too (Malaysia and Japan) to auto install.
The idea is that anyone could decide to use this format, even if they are not in Singapore. for example if their customer is from Singapore and they want to use the sg format to send invoice to them.